### PR TITLE
[Admin][Product] Removing images of a product

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php [hunk 1]
```diff
@@ -577,6 +594,52 @@ public function iChangeItsImageToPathForTheCode($path, $code)
...
+     */
+    public function thisProductShouldNotHaveImages()
+    {
+        /** @var UpdateSimpleProductPageInterface|UpdateConfigurableProductPageInterface $currentPage */
+        $currentPage = $this->currentPageResolver->getCurrentPageWithForm([
+            $this->updateSimpleProductPage,
+            $this->updateConfigurableProductPage,
+        ], $this->sharedStorage->get('product'));
+
+        Assert::eq(
```
Comment (tuka217) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php]: Should be `same` as `count` method return a number of elements in an array.

File: features/product/managing_products/removing_images_of_product.feature [hunk 2]
```diff
@@ -0,0 +1,101 @@
+@managing_products
+Feature: Removing images of an existing product
+    In order to remove images of my products
```
Comment (michalmarcinkowski) [inline features/product/managing_products/removing_images_of_product.feature]: In order to get rid of obsolete images of my products

File: features/product/managing_products/removing_images_of_product.feature [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        And this product should not have images
+
+    @ui @javascript
+    Scenario: Removing all images of a simple product
+        Given the store has a product "Lamborghini Gallardo Model"
+        And this product has an image "lamborghini.jpg" with a code "thumbnail"
+        And this product has also an image "lamborghini.jpg" with a code "main"
+        And I want to modify this product
+        When I remove an image with a code "thumbnail"
+        When I remove also an image with a code "main"
```
Comment (michalmarcinkowski) [inline features/product/managing_products/removing_images_of_product.feature]: `When`->`And`

File: features/product/managing_products/removing_images_of_product.feature [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        And this product should not have images
+
+    @ui @javascript
+    Scenario: Removing all images of a configurable product
+        Given the store has a "Lamborghini Gallardo Model" configurable product
+        And this product has an image "lamborghini.jpg" with a code "thumbnail"
+        And this product has also an image "lamborghini.jpg" with a code "main"
+        And I want to modify this product
+        When I remove an image with a code "thumbnail"
+        When I remove also an image with a code "main"
```
Comment (michalmarcinkowski) [inline features/product/managing_products/removing_images_of_product.feature]: `When` -> `And`

**Comment by michalmarcinkowski** [comment]
@GSadee needs rebase.

**Pull request merged**